### PR TITLE
TE-10998 Fixed GA

### PR DESCRIPTION
**Pull request by dmytro-dymarchuk**
RG: https://release.spryker.com/release-groups/view/4075

### CODE REVIEW

**Review by yaroslav-spryker — APPROVED** [review]

**Review by dimitriyTsemma — APPROVED** [review]

File: src/Silex/Provider/HttpFragmentServiceProvider.php [hunk 1]
```diff
@@ -97,4 +96,25 @@ public function boot(Application $app)
...
+    /**
+     * @return string
+     */
+    protected function getUriSignerSecret(): string
+    {
+        if ($this->uriSignerSecret) {
+            return $this->uriSignerSecret;
+        }
+
+        $this->uriSignerSecret = getenv('SPRYKER_ZED_REQUEST_TOKEN') ?: null;
```
Comment (demkos) [inline src/Silex/Provider/HttpFragmentServiceProvider.php:109]: This env variable is a part of Spryker, and cannot be used in SIlex lib.
Also, a fix for this module was already provided.
Please, read the expected behavior from the ticket carefully.

**Review by demkos — CHANGES_REQUESTED** [review]
I'm not working on this bugfix, so please remove my name from PR

**Review by demkos — COMMENTED** [review]
Please, provide the release table for this PR

**Comment by dmytro-dymarchuk** [comment]
> Please, provide the release table for this PR

These changes look like merge only. There are only changes in CI and tests which we usually don't describe in changelogs.

**Comment by floriankraemer** [comment]
This was removed from the original RG because this is no longer directly related to the ticket.

**Comment by asmarovydlo** [comment]
Closed because of https://github.com/spryker/silexphp/pull/41#issuecomment-1176152385
Can be re-opened if necessary and linked to the new ticket.

**Pull request closed without merging**